docs/unified/protectli/building-manual.md

```diff
@@ -0,0 +1,373 @@
+# Building manual
+
+=== "fw6"
+
+    ## Requirements
+
+    - Docker
+        + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+        + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
```

Comment:
If docker is common for all, does it have to be duplicated? Can we have a common section with requirements, not tied to any platform?